basis/ui/gadgets/theme/theme.factor

```diff
@@ -1,8 +1,97 @@
-! (c)2009, 2010 Slava Pestov, Joe Groff bsd license
+! Copyright (C) 2009, 2010 Slava Pestov, Joe Groff.
+! See http://factorcode.org/license.txt for BSD license.
 USING: io.pathnames sequences ui.images vocabs ;
 IN: ui.gadgets.theme
 
 : theme-image ( name -- image-name )
     "vocab:ui/gadgets/theme/" prepend-path ".tiff" append <image-name> ;
 
+SYMBOL: theme
+SINGLETON: light-theme
```

Comment:
Shouldn't these be defined in their respective ``ui.themes.light`` or ``ui.themes.dark`` vocabularies?